### PR TITLE
Create a PS script for local testing

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**
This PS script assumes the ClrInstrumentationEngine MSI is installed. It launches a user-specified application (or PowerShell if none specified) with the COR_PROFILER variables as well as any configuration variable set for testing purposes.

### CODE REVIEW

File: src/Scripts/WindowsLocalTest.ps1 [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    [Parameter(Mandatory = $false)]
+    [switch]
+    $ProxyUsePreview
+)
+
+# COR_PROFILER variables
+$env:COR_ENABLE_PROFILING = 1
+$env:COR_PROFILER = "{324F817A-7420-4E6D-B3C1-143FBED6D855}"
+$env:COR_PROFILER_PATH_32 = "C:\Program Files (x86)\Microsoft CLR Instrumentation Engine\Proxy\v1\InstrumentationEngine.ProfilerProxy_x86.dll"
```
Comment (wiktork) [inline src/Scripts/WindowsLocalTest.ps1]: $env:ProgramFiles(x86) etc?

File: src/Scripts/WindowsLocalTest.ps1 [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+# CLRIE variables
+if ($DebugWait) {
+    $env:MicrosoftInstrumentationEngine_DebugWait = 1
+} else {
+    $env:MicrosoftInstrumentationEngine_DebugWait = 0
+}
+
+if ($DisableSignatureValidation) {
+    $env:MicrosoftInstrumentationEngine_DisableCodeSignatureValidation = 1
```
Comment (wiktork) [inline src/Scripts/WindowsLocalTest.ps1]: IIRC we don't check the actual value, just the existence of the environment variable.

Comment (WilliamXieMSFT) [reply src/Scripts/WindowsLocalTest.ps1]: You are correct

---
In reply to: [330623962](https://github.com/microsoft/CLRInstrumentationEngine/pull/139#discussion_r330623962) [](ancestors = 330623962)

File: src/Scripts/WindowsLocalTest.ps1 [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        name   = 'CORECLR_PROFILER_PATH_32'
+        value  = "${env:ProgramFiles(x86)}\Microsoft CLR Instrumentation Engine\Proxy\v1\InstrumentationEngine.ProfilerProxy_x86.dll"
+        enable = $true
+    }
+    @{
+        name   = 'CORECLR_PROFILER_PATH_64'
+        value  = "$env:ProgramFiles\Microsoft CLR Instrumentation Engine\Proxy\v1\InstrumentationEngine.ProfilerProxy_x64.dll"
+        enable = $true
+    }
+    # CLRIE-specifie
```
Comment (WilliamXieMSFT) [inline src/Scripts/WindowsLocalTest.ps1]: >e [](start = 19, length = 1)

c

**Pull request merged**